src/nvim/mbyte.c

```diff
@@ -2188,192 +1735,11 @@ int utf_class(int c)
  */
 
 /*
- * The following tables are built by ../runtime/tools/unicode.vim.
- * They must be in numeric order, because we use binary search.
- * An entry such as {0x41,0x5a,1,32} means that Unicode characters in the
- * range from 0x41 to 0x5a inclusive, stepping by 1, are changed to
- * folded/upper/lower by adding 32.
```

Comment:
Can this comment be added to the generated tables, so the meaning of these tables is not "lost"?